src/Groups/homomorphisms.jl

```diff
@@ -517,6 +524,29 @@ function isomorphism(::Type{T}, G::GAPGroup) where T <: Union{FPGroup, PcGroup,
    end::GAPGroupHomomorphism{typeof(G), T}
 end
 
+# If `G` is not a full pc group then switch to a full pc group in Oscar.
```

Comment:
This, by the way, is something that I needed a lot in the past, and was always annoyingly complicated in GAP. So this is a nice bonus I did not even consider (at least I don't recall considering it :-) )

Reply:
I think GAP is just missing something like `IsomorphismFullPcGroup`.